sklego/preprocessing.py

```diff
@@ -403,6 +405,9 @@ def fit(self, X, y=None):
         Store the inverse of R of the QR decomposition of X, which can be used to calculate the orthogonal projection
         of X. If normalization is required, also stores a vector with normalization terms
         """
+        if type(X) is pd.core.frame.DataFrame:
```

Comment:
preferable: `isinstance(X, pd.DataFrame)`

Reply:
These are no more in code